assets/styledown.css

```diff
@@ -136,6 +136,8 @@ code.sg {
  */
 
 .sg-code {
+  max-width: 500px;
```

Comment:
how about using the viewport unit `vw`, so that the allowed width stretches with the available width of the display?

Reply:
🤔 maybe 20vw